setup/install_gatling/10_pe_install/10_install_pe.rb

```diff
@@ -1,6 +1,7 @@
 require 'beaker-pe-large-environments'
 
 test_name 'install PE for a scale environment' do
+  skip_test 'Installing FOSS, not PE' unless ENV['BEAKER_INSTALL_TYPE'].casecmp('pe') == 0
```

Comment:
this is kinda hard to read.  
maybe put it in another variable like
```
install_type_pe? = ENV['BEAKER_INSTALL_TYPE'].casecmp('pe') ? true : false
...
skip_test if install_type_pe?

Reply:
later, we should be using options files , or the rakefile to specify which pre-suite files get run in what order for different types of runs.